locale/en_US/submission.xml

```diff
@@ -332,12 +332,11 @@
 	<message key="submission.layout.confirmDeleteSupplementaryFile">Are you sure you want to permanently delete this supplementary file?</message>
 	<message key="submission.layout.deleteGalleyStylesheet">Delete this style sheet.</message>
 	<message key="submission.layout.editGalley">Edit a Layout Galley</message>
-	<message key="submission.layout.galleyFileData">Galley File Information</message>
 	<message key="submission.layout.galleyFormat">Galley Format</message>
 	<message key="submission.layout.galleyHTMLData">HTML Galley Files</message>
 	<message key="submission.layout.galleyImages">Images</message>
-	<message key="submission.layout.galleyLabelInstructions">The galley label is used to identify the item's file format (e.g. HTML, PDF, etc.).</message>
-	<message key="submission.layout.galleyLabel">Label</message>
+	<message key="submission.layout.galleyLabel">Galley Label</message>
+	<message key="submission.layout.galleyLabelInstructions">Typically used to identify the file format (e.g. PDF, HTML, etc.).</message>
```

Comment:
Hmm, I'm not sure about this. The English updates are less-than-critical updates to phrasing for clarity. I wonder if it's best keeping the existing translations for all these languages so close to a release, knowing that they are still essentially accurate and many might not update in time for OJS 3.0.

@asmecher or @mtub, do you have an opinion on this?